tests/integration/test_e2e.py

```diff
@@ -62,7 +63,7 @@ def test_multiple_singer_json_messages(self):
             {'tap_stream_name': 'dummy_stream', 'field_id': 'column_4', 'type': 'HASH-SKIP-FIRST-3'},
             {'tap_stream_name': 'dummy_stream', 'field_id': 'column_5', 'type': 'MASK-DATE'},
             {'tap_stream_name': 'dummy_stream', 'field_id': 'column_6', 'type': 'MASK-NUMBER'},
-            {'tap_stream_name': 'dummy_stream', 'field_id': 'column_7', 'type': 'NOT-EXISTING-TRANSFORMATION-TYPE'},
+            # {'tap_stream_name': 'dummy_stream', 'field_id': 'column_7', 'type': 'NOT-EXISTING-TRANSFORMATION-TYPE'},
```

Comment:
can we remove this or you want to add more test here as well?

`UnsupportedTransformationTypeException` seems like united tested [here](https://github.com/transferwise/pipelinewise-transform-field/pull/21/files#diff-4c99501228893efd717521fd5f5029dafdcc11d76d845c19806a1df88b91a4acR81)

Reply:
I can remove it, no plans on adding any tests

Reply:
Done https://github.com/transferwise/pipelinewise-transform-field/pull/21/commits/8e8d3c16f7acaea2f84b8e55290c8040b98dc9f4